tests/request/test_sanic.py

```
@@ -28,22 +28,23 @@ async def test_sanic_adapter(app: Sanic[Any, Any]) -> None:
 
-    @app.post("/test")  # type: ignore[misc]
-    async def handler(request: Request) -> HTTPResponse:
+    @app.post("/test/<item_id>")  # type: ignore[misc]
+    async def handler(request: Request, item_id: str) -> HTTPResponse:
         nonlocal adapter_result
         adapter_result = SanicHTTPRequestAdapter(request)
         return text("OK")
 
     # Test with form data and files
     app.asgi_client.cookies["session"] = "123"
     _, response = await app.asgi_client.post(
```

Ruff (0.15.9)

[warning] Unpacked variable `response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)